static/js/app.js

```diff
@@ -312,11 +309,9 @@ function showBannersIfNecessary() {
   const bg = dashboardBanners.querySelector(".banner-gradient-bg");
   const showBanner = (bannerEl) => {
     setTimeout(()=> {
-      if (!isRelayAddonInstalled()) {
-        bg.style.minHeight = "101px";
-        bannerEl.classList.remove("hidden");
-        dashboardBanners.classList.remove("invisible");
-      }
+      bg.style.minHeight = "101px";
```

Comment:
I believe this `min-height` bit of code is super legacy and can be removed. We migrated to a `display: sticky` layout for the header a few releases ago, making this not as necessary.

Reply:
48d127a